openedx/core/djangoapps/oauth_dispatch/dot_overrides/backends.py

```diff
@@ -4,10 +4,9 @@
 
 
 from django.contrib.auth.backends import AllowAllUsersModelBackend as UserModelBackend
-from ratelimitbackend.backends import RateLimitMixin
 
 
-class EdxRateLimitedAllowAllUsersModelBackend(RateLimitMixin, UserModelBackend):
```

Comment:
https://github.com/openedx/django-ratelimit-backend/blob/master/ratelimitbackend/backends.py

Reply:
Is there any other mixin require here ?

Reply:
Given that we're no longer mixing in the RateLimitMixin, we should just remove this class and use the `UserModelBackend` directly wherever the EdxRateLimitedAllowAllUsersModelBackend is being used?